doc/examples/integrations_all_in_one.py

```diff
@@ -2,8 +2,11 @@
 # -*- coding: utf-8 -*-
 
 from __future__ import absolute_import
```

Comment:
Is the `absolute_import` still required?

Reply:
Absolute import makes python use the import system that is in python 3, the specifics as to why it's necessary I don't have, but since these scripts are all python 2 compatible I figured it best to leave it.